src/CheckoutSdk/Payments/IPaymentsClient.cs

```diff
@@ -12,6 +12,6 @@ Task<PaymentResponse> RequestAsync<TPaymentSource>(PaymentRequest<TPaymentSource
         Task<CaptureResponse> CaptureAsync(string paymentId, CaptureRequest captureRequest = null, CancellationToken cancellationToken = default(CancellationToken));
         Task<RefundResponse> RefundAsync(string paymentId, RefundRequest refundRequest = null, CancellationToken cancellationToken = default(CancellationToken));
         Task<GetPaymentResponse> GetAsync(string paymentId, CancellationToken cancellationToken = default(CancellationToken));
-        Task<ICollection<ActionProcessed>> GetActionsAsync(string paymentId, CancellationToken cancellationToken = default(CancellationToken));
+        Task<ICollection<Action>> GetActionsAsync(string paymentId, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Should be `IEnumerable` not `ICollection`

Reply:
That way Resharper will complain about possible multiple enumerations.

Will fix in [this PR](https://github.com/CKOTech/checkout-sdk-net/pull/9) to minimize conflicts.